### PR TITLE
fix: [Spore] polish search

### PR DESCRIPTION
**Pull request by callil**
## Description
The current search box looks weird/bad. Update it to look better. 
- Smaller height and more borer radius
- Update hover outline to be more "uniswappy"

**todo: test on mobile**

## Screen capture

|    Before    |   After    |
| ------------ | ------------ |
| Screenshot 2023-09-11 at 12 17 37 PM | Screenshot 2023-09-11 at 12 17 26 PM |

### CODE REVIEW

File: src/nft/css/common.css.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ const magicalGradient = style({
       content: '',
       position: 'absolute',
       inset: '-1px',
-      background: 'linear-gradient(91.46deg, #4673FA 0%, #9646FA 100.13%) border-box',
+      background: 'linear-gradient(45deg, #FC72FF40 0%, #FC72FF20 100.13%) border-box',
```
Comment (cbachmeier) [inline src/nft/css/common.css.ts:31]: non-blocking: border is a very faint so it's hard to notice imo
Screenshot 2023-09-12 at 9 41 40 AM
Screenshot 2023-09-12 at 9 42 12 AM

File: src/components/NavBar/SearchBar.tsx [hunk 2]
```diff
@@ -163,11 +163,11 @@ export const SearchBar = () => {
             !isOpen && !isMobile && magicalGradientOnHover,
             isMobileOrTablet && (isOpen ? styles.visible : styles.hidden)
           )}
-          borderRadius={isOpen || isMobileOrTablet ? undefined : '12'}
-          borderTopRightRadius={isOpen && !isMobile ? '12' : undefined}
-          borderTopLeftRadius={isOpen && !isMobile ? '12' : undefined}
+          borderRadius={isOpen || isMobileOrTablet ? undefined : '16'}
+          borderTopRightRadius={isOpen && !isMobile ? '16' : undefined}
+          borderTopLeftRadius={isOpen && !isMobile ? '16' : undefined}
           borderBottomWidth={isOpen || isMobileOrTablet ? '0px' : '1px'}
```
Comment (cbachmeier) [inline src/components/NavBar/SearchBar.tsx:169]: border is missing on the bottom left and right corners when open

before:
Screenshot 2023-09-12 at 9 44 33 AM

after:
Screenshot 2023-09-12 at 9 38 34 AM

Comment (callil) [reply src/components/NavBar/SearchBar.tsx:169]: Fix'd

File: src/nft/css/common.css.ts (hunk 1 above)
Comment (callil) [reply src/nft/css/common.css.ts]: good point - brought to 80% opacity

**Review by cbachmeier — APPROVED** [review]
LGTM

**Review by cartcrom — APPROVED** [review]

**Pull request merged**